raft/src/main/java/org/apache/kafka/raft/KafkaRaftClient.java

```diff
@@ -2618,6 +2656,14 @@ private void handleResponse(RaftResponse.Inbound response, long currentTimeMs) {
                 handledSuccessfully = handleUpdateVoterResponse(response, currentTimeMs);
                 break;
 
+            case ADD_RAFT_VOTER:
+                handledSuccessfully = handleAddVoterResponse(response, currentTimeMs);
+                break;
+
+            case REMOVE_RAFT_VOTER:
+                handledSuccessfully = handleRemoveVoterResponse(response, currentTimeMs);
+                break;
```

Comment:
Since now kraft can send these RPCs, we need to add a case for them in `RaftUtil#errorResponse`.